plugins/polio/models.py

```diff
@@ -126,7 +126,7 @@ class RoundScope(models.Model):
     )
     round = models.ForeignKey("Round", on_delete=models.CASCADE, related_name="scopes")
 
-    vaccine = models.CharField(max_length=5, choices=VACCINES)
+    vaccine = models.CharField(max_length=5, choices=VACCINES, blank=True, null=True)
```

Comment:
I think we're trying to follow good practice and avoid using `null=True` for `CharField`: [django doc](https://docs.djangoproject.com/en/5.0/ref/models/fields/#null)

On the other hand we would have to check what the front-end sends

Reply:
> I think we're trying to follow good practice and avoid using `null=True` for `CharField`: [django doc](https://docs.djangoproject.com/en/5.0/ref/models/fields/#null)

I approve this message 😄

```suggestion
    vaccine = models.CharField(max_length=5, choices=VACCINES, blank=True)
```